src/nyaml/nyaml2nxdl.py

```diff
@@ -941,6 +994,8 @@ def xml_handle_fields_or_group(
     else:
         elemt_obj.set("name", keyword_name)
 
+    xml_handle_nametype(keyword, keyword_name, dct, elemt_obj)
+
```

Comment:
The same is here this utilized below L:1003, where we

Reply:
I see what your intention was here and in one of the previous comments, yeah agree, moved the calls respectively.

Reply:
done